components/brave_wallet/common/brave_wallet_types.h

```diff
@@ -9,12 +9,15 @@
 #include <string>
 #include <vector>
 
+#include "brave/third_party/boost/multiprecision/include/boost/multiprecision/cpp_int.hpp"
+
 namespace brave_wallet {
 
-typedef unsigned _BitInt(256) uint256_t;
-typedef _BitInt(256) int256_t;
-typedef unsigned _BitInt(128) uint128_t;
-typedef _BitInt(128) int128_t;
+typedef boost::multiprecision::uint256_t uint256_t;
+typedef boost::multiprecision::int256_t int256_t;
+typedef boost::multiprecision::uint128_t uint128_t;
```

Comment:
I guess I thought we'd better use both ints from the same place, but actually that shouldn't matter: a 128-bit integer should be the same thing regardless, otherwise we'd have bigger problems :grin: 

Fixed in https://github.com/brave/brave-core/pull/12384/commits/7f69717971b458cb53259e3ee49f145e4b91486b